includes/data-stores/abstract-wc-order-data-store-cpt.php

```diff
@@ -0,0 +1,260 @@
+<?php
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+/**
+ * Abstract Order Data Store: Stored in CPT.
+ *
+ * @version  2.7.0
+ * @category Class
+ * @author   WooThemes
+ */
+abstract class Abstract_WC_Order_Data_Store_CPT extends WC_Data_Store_CPT implements WC_Object_Data_Store, Abstract_WC_Order_Data_Store_Interface {
+	/*
+	|--------------------------------------------------------------------------
+	| CRUD Methods
+	|--------------------------------------------------------------------------
+	*/
+
+	/**
+	 * Method to create a new order in the database.
+	 * @param WC_Order $order
+	 */
+	public function create( &$order ) {
+		$order->set_version( WC_VERSION );
+		$order->set_date_created( current_time( 'timestamp' ) );
+		$order->set_currency( $order->get_currency() ? $order->get_currency() : get_woocommerce_currency() );
+
+		$id = wp_insert_post( apply_filters( 'woocommerce_new_order_data', array(
+			'post_date'     => date( 'Y-m-d H:i:s', $order->get_date_created( 'edit' ) ),
+			'post_date_gmt' => get_gmt_from_date( date( 'Y-m-d H:i:s', $order->get_date_created( 'edit' ) ) ),
+			'post_type'     => $order->get_type( 'edit' ),
+			'post_status'   => 'wc-' . ( $order->get_status( 'edit' ) ? $order->get_status( 'edit' ) : apply_filters( 'woocommerce_default_order_status', 'pending' ) ),
+			'ping_status'   => 'closed',
+			'post_author'   => 1,
```

Comment:
I notice that this was set to 1 before too. Is this something that doesn't really matter, even if user ID 1 is no longer a valid user?